end-to-end/artillery/queries/datasets.json

```diff
@@ -65,9 +65,9 @@
       "filter": { "term": { "entity_type.keyword": "Dataset" } },
       "aggs": {
         "donor.mapped_metadata.sex.keyword": {
-          "terms": { "field": "donor.mapped_metadata.sex.keyword", "size": 5 }
+          "terms": { "field": "donor..keyword", "size": 5 }
         },
-        "donor.mapped_metadata.sex.keyword_count": {
+        "donor..keyword_count": {
```

Comment:
I don't think you want to touch this file at all. Revert to original.

Reply:
I did not touch this file. This is how it looks on my computer. I do not know why it looks different after I push the changes to github, because I did not do anything to this file. 
 
![Screen Shot 2021-07-07 at 1 06 19 PM](https://user-images.githubusercontent.com/85570580/124800901-30793a80-df24-11eb-8b7e-f25b32bf077e.png)

Reply:
If that's the case, you can use `git pull` to pull down the current code and change it back and commit it afterwards.

Reply:
There are other ways to do it, but that's probably the most straightforward.